string driveName
int16 driveMode
float32[] drivecommands
string armName
float32[] armcommands
